src/utils/frontend-api-client/contractors.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios'
+
+export const getContractors = async (propertyReference, tradeCode) => {
+  const { data } = await axios.get(
```

Comment:
I've got a slight preference for using the params object for axios rather than a string with interpolated query params. One of the reasons is that we get to split it out over multiple lines which could be useful for git diffing.

This could be rewritten

```
  ...axios.get('/api/contractors', {
      params: {
        propertyReference,
        tradeCode
      }
  })
```